us.tylerc.cycles.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: us.tylerc.cycles
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
current runtime is 23.08
```suggestion
runtime-version: '23.08'
```

Reply:
This doesn't hurt compatibility with older OS's? It seemed unclear when I was looking.